UnitTests/GitUITests/UserControls/BlameControlTests.cs

```diff
@@ -5,13 +5,14 @@
 using System.Threading;
 using FluentAssertions;
 using GitCommands;
-using GitUI;
 using GitUI.Blame;
 using GitUIPluginInterfaces;
 using NUnit.Framework;
 
 namespace GitUITests.UserControls
 {
+    [SetCulture("en-US")]
```

Comment:
I like that (a lot more than the deleted lines just below)!

But I'm really not fond of the "en-US" culture chosen (rant: I really dislike their dumb and only used by them date format) but I'm not the center of the universe 😉.